### DISCUSSION TITLE
Firefox asking to save MySQL port/password

### DISCUSSION THREAD

**Discussion by diogenesbrussels** [Budibase/budibase#4753] [Help] [answered]
Hello,

I got this strange situation with Firefox:
- I'm logging in as administrator on a self-hosted Budibase instance.
- I open an application to modify it.
- I want to edit a table in a MySQL connection.
- The Firefox password save prompt appears to keep the username: 3306 and the database password.
![mysql firefox save password](https://user-images.githubusercontent.com/91942877/156162400-4c45a709-6c9f-454b-b858-c0bc405ca0d3.png)

Firefox 97.0.1
Budibase 1.0.73

This is not a problem for me but if I allow one of my "less technical" colleagues to change something he might be disturbed.

**Comment by mjashanks** [comment, marked as answer]
Thanks @diogenesbrussels - we'll get this fixed.
